src/andromede/libs/antares_historic/antares_historic.yml

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+library:
+  id: antares-historic
+  description: Antares historic model library
+
+  port-types:
+    - id: flow
+      description: A port which transfers power flow
+      fields:
+        - name: flow
+
+  models:
+    - id: area
+      parameters:
+        - name: spillage_cost
+        - name: ens_cost
+      variables:
+        - name: spillage
+          lower-bound: 0
+          type: float
```

Comment:
In Antares studies, there is a "fictive load" constraint on the upper bound on spillage

Reply:
We need ports to handle that, will treat it later on